### DISCUSSION TITLE
Requested Feature

### DISCUSSION THREAD

**Discussion by scubastan** [madMAx43v3r/chia-plotter#621] [Ideas]
Anyway to set Madmax to stop plotting after completing its current plot without continuing its loop?
I know you can hit Ctrl-C to kill the plot.. but sometimes it might be easier to hit a different command, and the current plot will finish and the loop will stop.

**Comment by wabragg** [comment]
That's the way it works now if you only hit a single Control-C.  If you double tap, it will terminate immediately without cleaning up after itself.

**Comment by andymarden** [comment]
Ctrl-c literally does exactly that
